README.md

```diff
@@ -51,27 +52,30 @@ for _ in range(1000):
 ```
 
 # Contents
-1. [Documentation](#Documentation)
-1. [Setup](#Setup)
-    + [Installation](#Installation)
-    + [Running](#Running)
-1. [Examples](#Examples)
-    + [Usage](#Usage)
-    + [Reinforcement Learning](#Reinforcement-Learning)
-1. [CLI Tool](#CLI-Tool)  
-    + [CLI Usage](#CLI-Usage)
-    + [CLI Examples](#CLI-Examples)
-1. [Visualizing Observations](#Visualizing-Observations)
-1. [PyMARL and MALib](#PyMARL-and-MALib)
-1. [Containers](#Containers)
-    + [Docker](#Docker)
-    + [Singularity](#Singularity)
-1. [Troubleshooting](#Troubleshooting)
-    + [General](#General)
-    + [SUMO](#SUMO)
-1. [Bug Reports](#Bug-Reports)
-1. [Contributing](#Contributing)
-1. [Citing](#Citing)
+- [SMARTS](#smarts)
+- [Multi-agent experiment as simple as ...](#multi-agent-experiment-as-simple-as-)
```

Comment:
Modified the link, as the original link doesn't seem to work.

```suggestion
- [Multi-agent experiment as simple as ...](#multi-agent-experiment-as-simple-as)
```

Reply:
This was an unintentional change made by my editor. I've reverted it back.